### PR TITLE
Project Initialization

### PR DESCRIPTION
**Pull request by fmartin1**
Create gitignore
Create multi-module maven project with 3 modules:

- poster-core: PTOs and other classes needed by the other modules
- poster-data: which will work as a micro service connected to the DB which will receive messages from RabbitMQ
- poster-webapp: which is the JSPs that receive and send messages to poster-data.

### CODE REVIEW

File: poster/poster-core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <artifactId>poster-core</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>poster-core</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
```
Comment (Joxebus) [inline poster/poster-core/pom.xml]: Use Java 8

File: poster/poster-core/pom.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <!-- https://mvnrepository.com/artifact/javax.persistence/persistence-api -->
+    <dependency>
+      <groupId>javax.persistence</groupId>
+      <artifactId>persistence-api</artifactId>
+      <version>1.0.2</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
```
Comment (Joxebus) [inline poster/poster-core/pom.xml]: Verify if all the plugins listed are required.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Compiled class file
```
Comment (Joxebus) [inline .gitignore]: Verify if all the ignored files are required.

File: poster/poster-data/.gitignore [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/target/
```
Comment (Joxebus) [inline poster/poster-data/.gitignore]: Verify if all the elements listed are required.

File: poster/poster-data/.mvn/wrapper/maven-wrapper.properties [hunk 5]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.3/apache-maven-3.5.3-bin.zip
```
Comment (Joxebus) [inline poster/poster-data/.mvn/wrapper/maven-wrapper.properties]: Remove maven wrapper and all files related.

File: poster/poster-core/pom.xml [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    <groupId>fmartinez</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>fmartinez</groupId>
+  <artifactId>poster-core</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>poster-core</name>
+  <!-- FIXME change it to the project's website -->
```
Comment (grunger94) [inline poster/poster-core/pom.xml]: Remove not needed comments

**Comment by grunger94** [comment]
Replace .properties files in favor of .yml

**Comment by fmartin1** [comment]
Thanks

**Pull request closed without merging**